web/client/translations/data.en-US.json

```diff
@@ -1127,14 +1127,19 @@
         },
         "mapExport": {
             "title": "Export Map",
+            "exportPanel": {
+                "title": "<p><h4>Select your format</h4></p>",
+                "exportButtonLabel": "Export"
+            },
             "formats": {
                 "mapstore2": {
-                    "label": "JSON",
-                    "tooltip": "Export the map to MapStore2 JSON format"
+                    "label": "MapStore JSON",
+                    "description": "<p><small>Fully supported MapStore map configuration format</small></p>"
```

Comment:
Remove "Fully supported" from here and the other translations. Now I see it's unclear. only "MapStore map format" is enough.